### ISSUE TITLE
Nan in training loss

### ISSUE THREAD

**Issue by tshiono88** [Y-debug-sys/Diffusion-TS#6]
Hi, I really appreciate your great contribution to time series generation. I tried your unconditional generation tutorial with changing to my own dataset. But, I sometimes face "nan" in training loss. I checked my dataset and the way of transformation does not cause nan, and nan occurrance looks occational. Do you have any idea why nan occurs in training loss.

**Comment by Y-debug-sys** [comment]
That is strange. we have not met such problem. Here are some potential solutions but not sure: 1) increase batch size 2) decrease grad_clip 3) disable use_ff=True in ./Models/interpretable_diffusion/gaussian_diffusion.py/Diffusion-TS if your data is irregular 4) disable amp (see issues https://github.com/lucidrains/denoising-diffusion-pytorch/issues/61), but this reason should be impossible because I use the default float32. Can you provide more config and dataset infomation? I'm trying to reproduce your result.

**Comment by Hikari-C** [comment]
I just met the same problem, have you solved this using the mentioned approach?
